docs/modules/ROOT/pages/references/style-guide.adoc

```diff
@@ -0,0 +1,266 @@
+= Style Guide
+
+The Jsonnet related parts of this guide build on top of https://github.com/databricks/jsonnet-style-guide/blob/30d4d54444c1001ba9561afd7c8de405630a2ca7/README.md[Jsonnet Guide] by https://databricks.com/[Databricks].
+
+http://creativecommons.org/licenses/by-nc-sa/4.0/[image:https://i.creativecommons.org/l/by-nc-sa/4.0/88x31.png[Creative Commons License]] +
+This work is licensed under a http://creativecommons.org/licenses/by-nc-sa/4.0/[Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License].
+
+== General Style
+
+* Encode text files with UTF-8.
+* Use Unixe line endings (lf).
+* Always end a file with a single empty line.
+  This doesn't apply to empty files (for example `.gitkeep`).
+* Use two spaces for indentation except for `Makefile` where a single tab is to be used.
+* Trim whitespaces from the end of a line.
+
+[NOTE]
+====
+Repositories of Project Syn provide an https://editorconfig.org/c[EditorConfig].
+When using a supported editor, space types and number of spaces will automatically be done correctly.
+====
+
+== Asciidoc Style
+
+* Don't split senctences across lines but keep them on one single line.
+* Start a new line for each and every sentence.
+
+== Jsonnet Style
+
+== Syntactic Style
+
+=== Auto Formatting
+
+Use `jsonnet fmt` to format files. This will fix basic style errors.
+
+=== Variable Declaration
+
+* Variables should be named in camelCase style, and should have self-evident names.
++
+[source,jsonnet]
+----
+local serverPort = 1000;
+local clientPort = 2000;
+----
+
+* Prefer `local` to `::` syntax for private/local variables.
+  Unlike `::`, variables defined with `local` can't be overridden by children, nor accessed by other files.
++
+[source,jsonnet]
+----
+{
+  // CORRECT
+  local myVariable = 3,
+  result: myVariable + 1,
+
+  // INCORRECT
+  myVariable:: 3,
+  result: $.myVariable + 1,
+}
+----
+
+=== Line Length
+
+* Limit lines to 120 characters.
+* The only exceptions are import statements and URLs (although even for those, try to keep them under 100 chars).
```

Comment:
In the line above we talk about 120 chars and here about 100. What's correct?

Reply:
Fixed: 120 is correct.